src/Sylius/Component/Review/Factory/ReviewFactory.php

```diff
@@ -31,7 +34,6 @@ public function createNew(): ReviewInterface
 
     public function createForSubject(ReviewableInterface $subject): ReviewInterface
     {
-        /** @var ReviewInterface $review */
         $review = $this->factory->createNew();
```

Comment:
Same change as for others?

```suggestion
        $review = $this->createNew();
```